src/CrossValidation.cpp

```diff
@@ -224,6 +265,15 @@ int CrossValidation::doStep(bool updateDOC, Normalizer* pNorm, double selectionF
   pOptions->cgitermax = CGITERMAX;
   pOptions->mfnitermax = MFNITERMAX;
   int estTruePos = 0;
+
+  // omp_set_dynamic(0);
```

Comment:
Can we somehow do this with ifdefs, so compilers without openmp support can also compile this (and run on a single core).

Reply:
Updated the code to only do this if openmp is supported.  Otherwise, everything is run using single-threading